opm/autodiff/BlackoilWellModel.hpp

```diff
@@ -69,6 +69,10 @@ namespace Opm {
             typedef WellStateFullyImplicitBlackoil WellState;
             typedef BlackoilModelParameters ModelParameters;
 
+            static const int Water = BlackoilPhases::Aqua;
```

Comment:
I suggest getting these from WellInterface instead, so there is only one definition instead of a duplication.

Reply:
Fixed in new PR